db/migrate/20230926153652_drop_unique_index_on_application_choice_form_and_course.rb

```diff
@@ -0,0 +1,11 @@
+class DropUniqueIndexOnApplicationChoiceFormAndCourse < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    remove_index :application_choices, %w[application_form_id course_option_id], name: 'index_course_option_to_application_form_id', if_exists: true
```

Comment:
Do we need concurrently to not lock the application choices table?

Reply:
Yes that's right.

https://medium.com/carwow-product-engineering/problems-with-concurrent-postgres-indexes-and-how-to-solve-them-c57f7656c852